src/xdocs/config_naming.xml

```diff
@@ -472,10 +472,14 @@ public class MyClass {
         </source>
         <p>Example:</p>
         <source>
-abstract class AbstractFirstClass {} // OK
-abstract class SecondClass {} // violation, it should match pattern "^Abstract.+$"
-class AbstractThirdClass {} // violation, must be declared 'abstract'
-class FourthClass {} // OK
+class Example1 {
```

Comment:
Was this just manually updated? I am not seeing any tests to ensure this matches input file.

Reply:
Reason at https://github.com/checkstyle/checkstyle/pull/13112#issuecomment-1573719786

Reply:
Then as I understand, this is just an example and needs to be removed for merging. It would be better for this to actually be done by the utility to help ensure the examples are transferred exactly.

Reply:
I think we are keeping those. And for 2 more checks as explained in https://github.com/checkstyle/checkstyle/pull/13112#discussion_r1215685075.

@romani please confirm

Reply:
I push team to do manually update for 3 Checks. To show final result, agree with all maintainers, and update all others without extended reviews.

This change will stay, be merged, be released/published far before final phase of this project.

If after automation there will be some minor corrections, it is ok.

Reply:
> If after automation there will be some minor corrections, it is ok.

The issue is the original will be gone, so the corrections will be on the modified and not the original. Its easier to reviewers to review the automated transformation then have to review the manual one and all its details.

Reply:
We took a lot of nuances from initial manual update. Such activity discovered a lot of items to agree on and put all on same page.
I recommend to finish 3 updates, and be sure we already agreed on 90% of points. 

This PR Update is valid, and can be merged just as regular doc improvement.

Reply:
Same argument as https://github.com/checkstyle/checkstyle/pull/13112#discussion_r1221850586 .

Reply:
Answered